pkg/controller/supportbundlecollection/validate.go

```diff
@@ -55,8 +65,17 @@ func (c *Controller) Validate(review *admv1.AdmissionReview) *admv1.AdmissionRes
 		return validationResult(allowed, msg)
 	}
 
-	if review.Request.Operation == admv1.Update {
+	switch review.Request.Operation {
+	case admv1.Create:
```

Comment:
Should build/charts/antrea/templates/webhooks/validating/crdvalidator.yaml be updated?

Reply:
Done, thanks
For some reason the webhook is registered for `DELETE`, even though it doesn't perform any checks for deletion. I didn't change it as part of this PR.